src/Common/AzurePSCmdlet.cs

```diff
@@ -434,6 +438,37 @@ protected bool IsVerbose()
             return verbose;
         }
 
+        protected void WriteSurvey()
+        {
+            HostInformationMessage msg = new HostInformationMessage()
```

Comment:
We can use more meaningful names
```suggestion
            HostInformationMessage surveyIntroMsg = new HostInformationMessage()
```